bin/generate_builtins.jl

```diff
@@ -2,16 +2,26 @@
 # Should be run on the latest Julia nightly
 using InteractiveUtils
 
-# All builtins present in 1.6
-const ALWAYS_PRESENT = Core.Builtin[
-    (<:), (===), Core._abstracttype, Core._apply_iterate, Core._apply_pure,
-    Core._call_in_world, Core._call_latest, Core._equiv_typedef, Core._expr,
-    Core._primitivetype, Core._setsuper!, Core._structtype, Core._typebody!,
-    Core._typevar, Core.apply_type, Core.ifelse, Core.sizeof, Core.svec,
-    applicable, fieldtype, getfield, invoke, isa, isdefined, nfields,
-    setfield!, throw, tuple, typeassert, typeof
+# All builtins present in 1.10
+const RECENTLY_ADDED = Core.Builtin[
```

Comment:
Why call this `RECENTLY_ADDED` instead of `ALWAYS_PRESENT`?

Reply:
The comment above was wrong. This list now contains builtins that have been added after 1.10.